src/hooks/spaces/useWorldAndSpaceBySlug.ts

```diff
@@ -41,7 +41,11 @@ export const useWorldAndSpaceBySlug = (
 
   const worldsRef = firestore
     .collection("worlds")
-    .where("slug", "==", worldSlug ?? "")
+    .where("isHidden", "==", false)
```

Comment:
I think less effort solutions might be `isVisible` or check `!= true` and leave the current state be same as when `undefined` is the value of the field (since it's being added as a new one)